src/const/urls.ts

```diff
@@ -16,6 +16,10 @@ export const JUMPER_WALLET_PATH = '/wallet/';
 export const JUMPER_FEST_PATH = '/superfest/';
 export const JUMPER_MEMECOIN_PATH = '/memecoins/';
 
+// jumper api endpoints
+export const JUMPER_ANALYTICS_EVENT = '/analytics/event';
```

Comment:
not fan of having frontend urls and backend in same file, easy to mix them up

Reply:
Alright, I have added `jumperApiUrls.ts` and put these two consts in there